src/azure-cli/azure/cli/command_modules/signalr/tests/latest/test_signalr_commands.py

```diff
@@ -17,24 +17,38 @@ def test_signalr_commands(self, resource_group):
         location = 'eastus'
         tags_key = 'key'
         tags_val = 'value'
+        service_mode = 'Classic'
+        allowed_origins = ['http://example1.com']
```

Comment:
I think it would be good to test a list. how about having at least 2 elements here?

Reply:
I'm using 2 elements in creating and updating cors.